src/index.jsx

```diff
@@ -3,7 +3,22 @@
 
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { positions, Provider } from 'react-alert';
+import AlertTemplate from 'react-alert-template-basic';
 import ViewManager from './ViewManager';
 import './index.css';
 
-ReactDOM.render(<ViewManager />, document.getElementById('root'));
+// place pop-up messages in bottom center of screen for 5s
+const options = {
+  timeout: 5000,
+  position: positions.BOTTOM_CENTER,
```

Comment:
Is it possible to get it in the bottom left corner instead? For me it is at the bottom and like 30px to the left